bench/wtperf/wtperf_truncate.c

```diff
@@ -54,6 +54,9 @@ setup_truncate(CONFIG *cfg, CONFIG_THREAD *thread, WT_SESSION *session) {
 	    session, cfg->uris[0], NULL, NULL, &cursor)) != 0)
 		goto err;
 
+	/* If we find the workload getting behind we multiply the stone gap */
```

Comment:
Good comment, but can you phrase it in terms of the number of records being truncated rather than "stone gap"?